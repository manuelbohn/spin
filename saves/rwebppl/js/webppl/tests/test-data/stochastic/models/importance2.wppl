var model = function() {
  var x = sample(Bernoulli({p: .5}), {guide() {
    return Bernoulli({p: .2})
  }});
  var y = sample(Bernoulli({p: .5}), {guide() {
    return Bernoulli({p: .7})
  }});
  factor((x || y) ? 0 : -Infinity);
  return [x, y];
};
