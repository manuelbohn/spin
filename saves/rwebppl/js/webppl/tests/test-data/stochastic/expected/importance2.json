{
  "hist": {
    "[true,false]": 0.33333333333333333,
    "[false,true]": 0.33333333333333333,
    "[true,true]": 0.33333333333333333
  },
  "logZ": -0.2876820724517809
}
